frontend/tsconfig.json

```diff
@@ -14,7 +14,8 @@
     "isolatedModules": true,
     "noEmit": true,
     "jsx": "react",
-    "baseUrl": "./"
+    "baseUrl": "./",
+    "noFallthroughCasesInSwitch": true
```

Comment:
Was this required for anything? Fine to leave it in, just curious

Reply:
I didn't see where that came from. I didn't make any intentional edits so this was automatic during the upgrade process.